client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -215,6 +215,25 @@ export default React.createClass( {
 		return null;
 	},
 
+	pendingGappsTosAcceptanceDomains() {
+		const domains = this.getDomains().filter( domain => domain.googleAppsSubscription && domain.googleAppsSubscription.hasPendingUsers );
+		if ( domains.length === 0 ) {
+			return null;
+		}
+
+		return (
+			<Notice
+				status="is-error"
+				showDismiss={ false }
+				key="pending-gapps-tos-acceptance-domains"
+				text={ this.translate( 'Urgent! You need to accept Google\'s Terms of Service or else your Google Apps accounts will get suspended soon.' ) }>
+				<NoticeAction href={ paths.domainManagementEmail( this.props.selectedSite.domain, this.getDomains().length === 1 && domains[0].name ) }>
+					{ this.translate( 'Fix now' ) }
```

Comment:
This breaks Calypso for me. `googleAppsUsers` has a null item in the state tree and breaks everything, including navigating away. 

![image](https://cloud.githubusercontent.com/assets/991022/15522426/1b3e4efa-21c7-11e6-9dec-74f8b374196a.png)